src/wp-includes/js/media/controllers/gallery-add.js

```diff
@@ -15,7 +15,7 @@ var Selection = wp.media.model.Selection,
  * @augments wp.media.controller.State
```

Comment:
Line 11: 'of' in @memberof shouldn't be capitalized. https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/#documenting-tips

Reply:
and move `memberof` to between `augments` and `param`